### PR TITLE
Update Helm installation instructions

### LINKED ISSUE #401: helm instructions should not include cloning the repo
**Description**
A brief description of the issue.

**Expected Behavior**
A brief description of what you expected to happen.

**Screenshots**
Add screenshots, if applicable, to help explain your problem.

**Enviroment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
[Optional] **To Reproduce**

[Optional] **Additional Context**

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

This PR fixes #401

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@alphaX86 @metonymic-smokey would you like to review?

File: _includes/getting-started.html [hunk 1]
```diff
@@ -87,54 +87,37 @@ <h3>Kubernetes User:</h3>
...
-kubectl create namespace meshery
+                  data-clipboard-text="kubectl create namespace meshery
 helm install meshery --namespace meshery install/kubernetes/helm/meshery" onmouseout="resetCopyText(this)">
                   <i class="far fa-copy"></i>
                   <span class="tooltiptext" style="font-size: 15px; width: 140px; height: 40px; padding: 0; line-height: 40px; top: 3rem; left: -20px;">Copy to clipboard</span>
                 </a>
               </div>
 
-            <p>Helm v2 - run the following:</p>
+            <!-- <p>Helm v2 - run the following:</p>
```
Comment (Aisuko) [inline _includes/getting-started.html:102]: We can remove the content with Helm v2

**Review by Aisuko — APPROVED** [review]
LGTM

**Comment by alphaX86** [comment]
I'd love to review this one too

**Comment by metonymic-smokey** [comment]
I'd love to review this PR.

File: _includes/getting-started.html [hunk 2]
```diff
@@ -87,54 +87,37 @@ <h3>Kubernetes User:</h3>
 
             <!-- Kubernetes - Helm -->
             <h4 style="font-size: 1.5em;">Using Helm</h4>
-            <p>Helm v3 - run the following:</p>
             <div id="start-helm-v3" class="highlight-code">
-              <p class="code-box" style="height: 90px; line-height: 30px;"><code style="position: absolute;"> $ git clone https://github.com/layer5io/meshery.git; cd meshery
-                $ kubectl create namespace meshery
-                $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
-                </code></p>
+              <p class="code-box" style="height: 60px; line-height: 30px;"><code style="position: absolute;">$ kubectl create namespace meshery
```
Comment (alphaX86) [inline _includes/getting-started.html:91]: The updated one looks great than the old one!

**Review by alphaX86 — APPROVED** [review]
LGTM so far! The changes are great! 👍

**Review by metonymic-smokey — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you much

File: _includes/getting-started.html (hunk 1 above)
Comment (leecalcote) [reply _includes/getting-started.html:102]: Oh, very good. No need to keep this old code in comments. Ok. Makes sense.

**Pull request merged**